src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -139,12 +178,203 @@ class IndexLogEntryTest extends SparkFunSuite {
             .Columns(Seq("col1"), Seq("col2", "col3")),
           schema.json,
           200)),
-      Content("rootContentPath", Seq()),
+      Content(Directory("rootContentPath")),
       Source(SparkPlan(expectedSourcePlanProperties)),
       Map())
     expected.state = "ACTIVE"
     expected.timestamp = 1578818514080L
 
+    val actual = JsonUtils.fromJson[IndexLogEntry](jsonString)
+    assert(actual.equals(expected))
+  }
+
+  test("Content Test: files api lists all files from Content object.") {
+    val dir2Path = PathUtils.makeAbsolute(nestedDir.toString)
```

Comment:
nit: what's the significance of 2 here? Either `dirPath` or `nestedDirPath` similar to how it's used in other places?

Reply:
I looked thru this and can you just use `testDirPath` and `nestedDirPath` throughout? I see `testDirLeafFiles`, etc. so let's make them consistent to make it easy to understand.

Reply:
thanks @imback82 , fixed